Documentation/community/roadmap.rst

```diff
@@ -164,8 +164,7 @@ Tetragon Security
 CLI for things like process execution, file access, network observability, and
 privileged execution.
 
-Although Tetragon is still in Beta stage, a set of adopters already use it in
-production.
+Tetragon is already used in production by adopters such as Github, Bell CA, Palantir, GResearch etc.
```

Comment:
I think we should just leave this sentence out from this page, since we don't talk about public adopters for other stable features 

```suggestion
```

Reply:
Sounds good, thanks.